src/jobservice/job/impl/gc/garbage_collection.go

```diff
@@ -728,6 +749,34 @@ func (gc *GarbageCollector) shouldStop(ctx job.Context) bool {
 	return false
 }
 
+// deletedArtBlobsInTimeRange queries the blobs associated with the artifact be deleted in the fixed time range.
+func (gc *GarbageCollector) deletedArtBlobsInTimeRange(ctx job.Context, start, end time.Time) ([]*blobModels.Blob, error) {
```

Comment:
The name deletedArtBlobsInTimeRange can be a bit confusing since it actually performs a listing action. Perhaps ListTrashedBlobs() would be a better name.

Reply:
make sense, will update.